src/base/utils/net.cpp

```diff
@@ -38,5 +40,42 @@ namespace Utils
         {
             return !QHostAddress(ip).isNull();
         }
+
+        bool isValidSubnet(const Subnet &subnet)
```

Comment:
>Where should this function be moved to? This code is used in multiple places so I think it makes sense as a net util.

You use it to check result of `QHostAddress::parseSubnet()` only so you will only need to do the following:
```c++
if (!QHostAddress::parseSubnet(subnetStr).first.isNull()) {
    // do something with valid subnet...
}
```
Or change this function to handle all possible invalid subnets and leave it here...

Reply:
I've renamed the function so it more accurately reflects the intended purpose.